sauna/package-lock.json

```diff
@@ -1,2935 +1,8 @@
 {
   "name": "sauna",
   "version": "0.0.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Update your npm installation with `npm install -g npm@latest` and reinstall packages.